libs/ui/src/components/NeoCollapsible/NeoCollapsible.scss

```diff
@@ -29,7 +29,7 @@
   .card-header-title {
     font-weight: 700;
     padding: 0.75rem;
-    margin: 0;
+    margin: 0 -3rem 0 0;
```

Comment:
this could have side effect on other components that use `NeoCollapsible`. have you check them?

Reply:
<img width="415" alt="image" src="https://github.com/kodadot/nft-gallery/assets/16473062/3dd82aa8-4bf9-4316-b405-aef5ff5b6078">

as i checked other pages used `NeoCollapsible` like the keyboard shorts setting page, it also has the misalignment issue  i believe it's a common fix. the reason that add `-3rem` margin here is due to the right arrow icon, it drift the text to the left of `3rem`

Reply:
to summarize this hotfix allows you to shift text at the center of collapsible, but from what I can see this is comming from `card-header`

btw content header is still missplaced for me (same for Collection Dropdown) 
![Screenshot 2023-09-14 at 09-25-46 Massmint KodaDot - NFT Market Explorer](https://github.com/kodadot/nft-gallery/assets/9987732/cf5b7245-ee3f-4cb9-8731-b4ef32d2b71a)


IMO this should have been aligned to left at first, wdyt @kodadot/design-squad ?

Reply:
> btw content header is still missplaced for me (same for Collection Dropdown)

<img width="509" alt="image" src="https://github.com/kodadot/nft-gallery/assets/16473062/08f63a7f-799c-46c9-b670-224acb7e8e19">

hmm wired... i see the text is centered. can you please help to check if the `margin` works?

Reply:
also i see there's a placeholder icon in the right of the first 2 header. so it a bit left aligned.

https://github.com/kodadot/nft-gallery/blob/be7b85d9eeec6eeea7e9a32a8d2867fd019ab27f/components/massmint/uploadDescription/UploadDescription.vue#L11

<img width="498" alt="image" src="https://github.com/kodadot/nft-gallery/assets/16473062/f382ec11-9081-4428-847d-cb7148f13ba8">